insights/parsers/cups_ppd.py

```diff
@@ -0,0 +1,52 @@
+"""
+CupsPpd - files ``/etc/cups/ppd/*``
+===================================
+
+Parser to parse the content of files ``/etc/cups/ppd/*``
+"""
+
+from insights import Parser
+from insights import parser
+from insights.specs import Specs
+
+
+@parser(Specs.cups_ppd)
+class CupsPpd(Parser):
+    """
+    Class to parse ``/etc/cups/ppd/*`` files.
+
+    Sample output for files::
+
+        *PPD-Adobe: "4.3"
+        *FormatVersion: "4.3"
+        *FileVersion: "2.2"
+        *LanguageVersion: English
+        *LanguageEncoding: ISOLatin1
+        *cupsFilter2: "application/vnd.cups-pdf application/pdf 10 -"
+        *cupsFilter2: "application/vnd.cups-postscript application/postscript 10 -"
+        *cupsFilter2: "application/vnd.cups-banner application/vnd.cups-banner 10 -"
+
+    Examples:
+        >>> type(cups_ppd)
+        <class 'insights.parsers.cups_ppd.CupsPpd'>
+        >>> cups_ppd["PCFileName"]
+        '"ippeve.ppd"'
+        >>> cups_ppd["cupsFilter2"]
+        ['"application/vnd.cups-pdf application/pdf 10 -"', '"application/vnd.cups-postscript application/postscript 10 -"']
+    """
+    def parse_content(self, content):
+        self.data = {}
```

Comment:
@wushiqinlou since you are only created a `dict` as the attribute, just go ahead and make the base class `dict` and then you'll get all of the methods for free.  [Here's an example](https://github.com/RedHatInsights/insights-core/blob/master/insights/parsers/hammer_ping.py) of how you can implement the parser as a `dict`.

Reply:
@bfahr - Updated, please have a check.